src/assets/TemplateFactory.ts

```diff
@@ -499,5 +499,94 @@ export const TemplateFactory = {
         return ret;
       })()
     };
+  }),
+  Colorfy: createTemplate((image: ImageSource, color: string) => {
+    return {
+      steps: [
+        {
+          image: [
+            {
+              source: new TemplateImage(image),
+              x: 0,
+              y: 0
+            }
+          ]
+        },
+        {
+          preprocess(canvas, ctx) {
+            ctx.fillStyle = color;
```

Comment:
```suggestion
            ctx.globalCompositeOperation = 'color';
            ctx.fillStyle = color;
```

`ctx.globalCompositeOperation = 'color';` is required for this to work properly